boards/BMS/Src/App/App_Accumulator.c

```diff
@@ -157,3 +158,21 @@ struct InRangeCheck *App_Accumulator_GetSegment5VoltageInRangeCheck(
 {
     return accumulator->segment_5_voltage_in_range_check;
 }
+
+bool App_Accumulator_IsMinCellVoltageOutOfRange(
```

Comment:
Within `App_SetPeriodicSignals_AccumulatorInRangeChecks` there are calls to `App_SetPeriodicCanSignals_InRangeCheck`. If we get this function to return its `InRangeCheck_Status` we will not have to check with the CAN struct whether something is out of range or not

Reply:
Would also avoid the need to add two more functions here to check if the mix and max cell voltages are out of range

Reply:
For relevant min and max voltages, we can do a simple check as well as set the error table in `App_SetPeriodicSignals_AccumulatorInRangeChecks` 

```
if(App_SetPeriodicCanSignals_InRangeCheck(...) != VALUE_IN_RANGE)
{
    set_error_table();
}
```